breathecode/admissions/tests/urls/tests_academy_cohort_id.py

```diff
@@ -672,3 +671,24 @@ def test_academy_cohort_id__with_data__testing_cache_and_remove_in_delete__2(sel
 
         AcademyCohortTestSuite.test_academy_cohort__with_data(self, base)
         self.assertEqual(self.cache.keys(), cache_keys)
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_academy_cohort_id__delete__cohort_with_students(self):
+        self.headers(academy=1)
+        model = self.generate_models(authenticate=True, profile_academy=True, 
+            capability='crud_cohort', role='potato' ,cohort_user=True)
+        url = reverse_lazy('admissions:academy_cohort_id', kwargs={'cohort_id': 1})
+        response = self.client.delete(url)
+        json = response.json()
+        expected = {
+            'detail': 'cohort-has-students',
+            'status_code': 400,
+        }
+        
+        self.assertEqual(self.count_cohort_user(), 1)
```

Comment:
The db test should be better for this way https://github.com/breatheco-de/apiv2/blob/2317ffb2f8ce08ee1623966657aa0939197ac94c/breathecode/admissions/tests/urls/tests_academy_cohort_id.py#L74

Reply:
You can copy paste it